onchain/rollups/test/foundry/dapp/CartesiDApp.t.sol

```diff
@@ -702,15 +702,28 @@ contract CartesiDAppTest is TestBase {
         );
     }
 
+    function encodeVoucher(
+        Voucher calldata voucher
+    ) external pure returns (bytes memory) {
+        return
+            OutputEncoding.encodeVoucher(voucher.destination, voucher.payload);
+    }
+
+    function encodeNotice(
+        bytes calldata notice
+    ) external pure returns (bytes memory) {
+        return OutputEncoding.encodeNotice(notice);
+    }
+
     function writeInputs() internal {
         for (uint256 i; i < outputEnums.length; ++i) {
             LibServerManager.OutputEnum outputEnum = outputEnums[i];
             if (outputEnum == LibServerManager.OutputEnum.VOUCHER) {
                 Voucher memory voucher = getVoucher(i);
-                writeInput(i, voucher.destination, voucher.payload);
+                writeInput(i, noticeSender, this.encodeVoucher(voucher));
```

Comment:
**Note:** This is a temporary solution to generate correct proofs. The new echo DApp should allow us to generate whatever vouchers and notices we need, as well as any output we might come up with in the future.